block/manager.go

```diff
@@ -30,10 +31,15 @@ import (
 // defaultDABlockTime is used only if DABlockTime is not configured for manager
 const defaultDABlockTime = 30 * time.Second
 
+// defaultBlockTime is used only if BlockTime is not configured for manager
+const defaultBlockTime = 1 * time.Second
+
 // maxSubmitAttempts defines how many times Rollkit will re-try to publish block to DA layer.
 // This is temporary solution. It will be removed in future versions.
 const maxSubmitAttempts = 30
 
+const channelLength = 100
```

Comment:
[non-blocking]  which channel? and is there an reason behind 100?

Reply:
added a comment explaining it:

> // Applies to all channels, 100 is a large enough buffer to avoid blocking